### DISCUSSION TITLE
Cant verify my account

### DISCUSSION THREAD

**Discussion by 0xmcha** [vercel/community#3671] [Account Issue] [answered]
### Question

Hello, im from Iraq. And when i try to create an account on vercel i get internal error when i try to verify my account with my phone number. 
![8489CD8F-D9B9-470D-8630-ABAB40D13C73](https://github.com/vercel/community/assets/110252456/76601a5b-c6dc-412f-a12b-7f8975556d93)

**Comment by amyegan** [comment, marked as answer]
Please email [registration@vercel.com](mailto:registration@vercel.com) and include the email and username for the account. Vercel employs multiple layers of security and verification. Providing additional information about your account and the project you are deploying can help accelerate this process.

**Reply by 0xmcha** [reply]
They solved my issue, may i ask why this happens? even my friend wants to register he get the same error.
